lib/semantic.rb

```diff
@@ -0,0 +1,7 @@
+module Semantic
+  autoload :Version, 'semantic/version'
```

Comment:
Since this is Semantic code this is an FYI and not a "this needs to be fixed", but Matz indicated that autoload is [deprecated](https://www.ruby-forum.com/topic/3036681) and should be avoided.